usaspending_api/api_docs/markdown/using_the_api.md

```diff
@@ -251,6 +247,24 @@ Field names support Django's foreign key traversal; for more details on this see
 #### POST Requests <a name="post-requests"></a>
 The structure of the post request allows for a flexible and complex query.
 
+#### POST Request Preservation <a name="post-requests-preservation"></a>
+All requests will return a `req` object in their response. This code allows you to share and preserve POST requests from page to page. You can send a request to any endpoint and specify the `req` from any request to re-run that request. For example, if you sent a POST request to `/api/v1/awards/` and returned a `req` of `abcd` you could re-run that request by hitting:
+
+`/api/v1/awards/?req=abcd`
+
+Or by POSTing
+```
+{
+  "req": "abcd"
+}
+```
+
+Do note that this _ignores_ pagination variables, so you can request a different page from the same request without changing your `req`. For example:
+
+`/api/v1/awards/?req=abcd&page=2`
+
```

Comment:
Good idea! I'm also going to add a specific tutorial about using the request checksums, etc.